### PR TITLE
set context steps.<id>.conclusion

### PR DESCRIPTION
**Pull request by ericsciple**
related to #274

### CODE REVIEW

File: src/Runner.Worker/ExecutionContext.cs [hunk 1]
```diff
@@ -309,6 +309,12 @@ public TaskResult Complete(TaskResult? result = null, string currentOperation =
                 Result = result;
             }
 
+            // update context
```
Comment (ericsciple) [inline src/Runner.Worker/ExecutionContext.cs]: still testing e2e

**Comment by MOZGIII** [comment]
This looks like something I could use - especially making the action cancelled part. Any plans to move forward with this?

**Comment by ericsciple** [comment]
@MOZGIII working on this soon. Hopefully rolled out within the next few weeks.

**Comment by ericsciple** [comment]
Closing this PR. Ting is working on it: https://github.com/actions/runner/pull/372

**Pull request closed without merging**